tests/endtoend/test_third_party_http_functions.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import os
+from unittest.mock import patch
+
+import requests
+
+from azure_functions_worker import testutils as utils, testutils
+
+# from tests.stein_tests import testutils
+# from tests.stein_tests.constants import E2E_TESTS_ROOT
+from azure_functions_worker.testutils import E2E_TESTS_ROOT
+
+HOST_JSON_TEMPLATE = """\
+{
+    "version": "2.0",
+    "logging": {
+        "logLevel": {
+           "default": "Trace"
+        }
+    },
+    "extensions": {
+        "http": {
+          "routePrefix": ""
+        }
+    },
+    "functionTimeout": "00:05:00"
+}
+"""
+
+
+class ThirdPartyHttpFunctionsTestBase:
```

Comment:
Why add this as a base class? This can be removed to keep only TestThirdPartyHttpFunctions

Reply:
pytest scans for all top level class with test methods, need to define this inner class to make pytest not scanning for base class tests

Reply:
pytest scans for all top level class with test methods, need to define this inner class to make pytest not scanning for base class tests